src/main/java/com/ibm/crypto/plus/provider/XDHPrivateKeyImpl.java

```diff
@@ -95,7 +95,7 @@ public XDHPrivateKeyImpl(OpenJCEPlusProvider provider, byte[] encoded)
         try {
             byte[] alteredEncoded = processEncodedPrivateKey(encoded); // Sets params, key, and algid, and alters encoded
             // to fit with GSKit and sets params
-            int priv_size = CurveUtil.getPrivateCurveSize(curve);
+            int priv_size = CurveUtil.getDEREncodingSize(curve);
```

Comment:
Should we change the name of the variable to `encodingSize`?

Reply:
Yes.